common_benchmark_suite/openxla/benchmark/comparative_suite/jax/test_data_definitions.py

```diff
@@ -58,6 +58,6 @@
                     "https://storage.googleapis.com/iree-model-artifacts/jax/jax_models_0.4.10_1684396752/T5_LARGE/batch_${batch_size}/output_npy.tgz"
                 ))
     })
-T5_LARGE_FP32_JAX_512X1024XF32_BATCHES = utils.build_batch_model_test_data(
+T5_LARGE_FP32_JAX_512X1024XF32_OUTPUT_BATCHES = utils.build_batch_model_test_data(
```

Comment:
For models, I think we've been following the convention: <model>_<data_type>_<framework>_<input_size>. For input and output data, I'm wondering if it should be prefixed with DATA_<input/output>_<model>_<data_type>_<framework>_<size>?